test/Numerics/DGmethods/compressible_Navier_Stokes/ref_state.jl

```diff
@@ -84,15 +84,15 @@ function run2(mpicomm, ArrayType, dim, topl, N, timeend, FT, dt)
         polynomialorder = N,
     )
 
-    T_min, T_s, Γ = FT(290), FT(320), FT(6.5 * 10^-3)
-    RH = 0.01
+    RH = FT(0.01)
```

Comment:
set `RH=0` here; otherwise, depending how this ref_state is being used, there may be energetic inconsistencies.